libs/datasets/dataset_orchestrator.py

```diff
@@ -0,0 +1,318 @@
+import json
+import datetime
+import structlog
+import pandas as pd
+from dataclasses import dataclass
+from typing import Dict, Iterable, List, Optional, Mapping
+
+from libs import pipeline
+from libs.parallel_utils import parallel_map
+from libs.datasets.sources import zeros_filter
+from libs.us_state_abbrev import US_STATE_ABBREV, ABBREV_US_UNKNOWN_COUNTY_FIPS
+from libs.datasets.timeseries import MultiRegionDataset
+from datapublic.common_fields import CommonFields, PdFields, FieldName
+from libs.datasets.dataset_utils import (
+    REGION_OVERRIDES_JSON,
+    CUMULATIVE_FIELDS_TO_FILTER,
+    DEFAULT_REPORTING_RATIO,
+)
+from libs.datasets import (
+    custom_patches,
+    weekly_hospitalizations,
+    nytimes_anomalies,
+    custom_aggregations,
+    statistical_areas,
+    timeseries,
+    outlier_detection,
+    dataset_utils,
+    combined_datasets,
+    new_cases_and_deaths,
+    vaccine_backfills,
+    tail_filter,
+    custom_patches,
+    weekly_hospitalizations,
+    manual_filter,
+    dataset_utils,
+    timeseries,
+    combined_datasets,
+)
+
+TailFilter = tail_filter.TailFilter
+_log = structlog.get_logger()
+
+# Putting this in dataset_utils caused a circular import with libs.pipeline
```

Comment:
This comment could use a bit more context, like, "Ideally this should probably go in dataset_utils, but that causes a circular import with libs.pipeline, so leaving here for now."